packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -306,11 +306,26 @@ class RouteSettings {
   /// The name of the route (e.g., "/settings").
   ///
   /// If null, the route is anonymous.
+  ///
+  /// The is also used to create a default semantic value for the route
+  /// transition used by accessibility frameworks. For example, "home/abc" will
+  /// be read as "home, abc" by TalkBack. On iOS, the route name is ignored by
+  /// default.
+  ///
+  /// See also:
+  ///
+  ///   * [Semantics.route]
```

Comment:
This should say why I should look at Semantics.route.

Reply:
Done